pkg/cmd/configs_tokens.go

```diff
@@ -137,13 +138,30 @@ func revokeConfigsTokens(cmd *cobra.Command, args []string) {
 
 	utils.RequireValue("token", localConfig.Token.Value)
 
-	slug := cmd.Flag("slug").Value.String()
+	slugFlagUsed := cmd.Flag("slug").Value.String() != ""
```

Comment:
Should we update our usage text to indicate that the param can be either a slug or token? Perhaps:

```diff
diff --git a/pkg/cmd/configs_tokens.go b/pkg/cmd/configs_tokens.go
index e898161..3f574f7 100644
--- a/pkg/cmd/configs_tokens.go
+++ b/pkg/cmd/configs_tokens.go
@@ -51,7 +51,7 @@ var configsTokensCreateCmd = &cobra.Command{
 }
 
 var configsTokensRevokeCmd = &cobra.Command{
-	Use:               "revoke [slug]",
+	Use:               "revoke [slug_or_token]",
 	Aliases:           []string{"delete"},
 	Short:             "Revoke a service token from a config",
 	Args:              cobra.MaximumNArgs(1),


```